.gitignore

```diff
@@ -1,4 +1,3 @@
 .DS_Store
 .vscode
-/web/private
```

Comment:
web private must be ignored because i use it for production configuration in my local environment

Reply:
There is another gitignore file inside that folder, that it's needed for reasons that I'll add below